pkg/api/v1beta3/types.go

```diff
@@ -990,7 +990,7 @@ type PodTemplateList struct {
 // ReplicationControllerSpec is the specification of a replication controller.
 type ReplicationControllerSpec struct {
 	// Replicas is the number of desired replicas.
```

Comment:
Would be helpful for comment to mention that it's a pointer to distinguish between explicitly zero and unspecified.